### PR TITLE
Fix text overflowing

### LINKED ISSUE #144: [PE-D][Tester E] Fields that cannot be read when GUI is in a small window
Perhaps would be good to wrap the text so that longer fields can still be readable on smaller app windows.
Also when the name is too long, the tag can't be read either.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: waynezsy/ped#3

### LINKED ISSUE #146: [PE-D][Tester D] Long positions and emails are cut off
When typing adding internship applications with long emails or long positions, these fields are cut off at the end. This might potentially be an important bug to fix considering certain positions have a slightly longer name such as "Summer 2023 Quantitative Research Intern".

However, this is not a common occurrence and thus the severity selected for this bug is low.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: seanmanik/ped#1

### LINKED ISSUE #152: [PE-D][Tester D] Location for Interviews cut off
When adding locations for interviews, locations with longer addresses are cut off at the end. Given that certain locations in Singapore can have long addresses, this might be a problem for users. However, this is also a relatively rare occurrence, thus severity is rated as low.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: seanmanik/ped#2

### LINKED ISSUE #156: [PE-D][Tester B] Interview text is not shown fully
Unable to see full text of route if it is a long string.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: jovitaanderson/ped#4

### LINKED ISSUE #162: [PE-D][Tester B] Long application company name string not handled
When the compnay name string is too long, it is not handled and i am unable to view the full text. In addition, the status is not printed properly as well.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: jovitaanderson/ped#6

### LINKED ISSUE #175: [PE-D][Tester C] Small UI error of Interview cards.
Should not have a space between "Google" and ","

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: maxtance/ped#11

### LINKED ISSUE #182: [PE-D][Tester E] Field name disappears when field is too long
The field name disappeared when I made the field even longer. (Position and Contact)

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: waynezsy/ped#8

### LINKED ISSUE #183: [PE-D][Tester E] Long tags makes status disappear
Interestingly `Position` reappeared.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: waynezsy/ped#9

### LINKED ISSUE #184: [PE-D][Tester E] Some of my tags are missing
Message from Ui:
```Edited Application: hi; Position applied: 111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111; Contact number: 11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111; Email: lowtierstudent@nus.edu.sg; Apply on: Aug 08 2021; Tags: [77777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777777555555][3333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333333][555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555555][66666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666666][222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222222][44444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444444][11111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111]```

As from picture, some of my tags are missing even though Ui tells me they were added.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: waynezsy/ped#10

### PR DESCRIPTION
**Pull request by jhchee18**
- Allow wrap text in every fields
- Fix status length to a min value
- Fix field label length to a min value
- Limit tag character length to 30
- Resize window min width and height to 800 (to prevent user from shrinking the window to very small then blame they cant see the word clearly on us...)
- Add line spacing and underline between fields to easier to distinguish between them after text wrapping

Fixed UI:
image

Resolves:
- #162
- #156
- #152
- #146
- #144
- #182 
- #183
- #184
- #175

### CODE REVIEW

File: src/main/java/seedu/application/ui/InterviewCard.java [hunk 1]
```diff
@@ -46,8 +49,8 @@ public InterviewCard(Application application, int displayedIndex) {
         super(FXML);
         this.interview = application.getInterview().get();
         id.setText("i" + displayedIndex + ". ");
-        company.setText(application.getCompany().company);
-        position.setText(application.getPosition().value);
+        companyAndPosition.setText(application.getCompany().company + ", " + application.getPosition().value);
+        //position.setText(application.getPosition().value);
```
Comment (jolynloh) [inline src/main/java/seedu/application/ui/InterviewCard.java]: same question as above ~

File: src/main/java/seedu/application/ui/InterviewCard.java [hunk 2]
```diff
@@ -27,9 +27,12 @@ public class InterviewCard extends UiPart<Region> {
     @FXML
     private Label id;
     @FXML
-    private Label company;
+    private Label companyAndPosition;
+    /*
     @FXML
     private Label position;
+
+     */
```
Comment (jolynloh) [inline src/main/java/seedu/application/ui/InterviewCard.java]: Is this intentionally commented out or can it be deleted?

**Review by jolynloh — COMMENTED** [review]
Just a minor question, other than that LGTM! Thanks for helping to resolve the issues 👍

File: src/main/java/seedu/application/ui/InterviewCard.java (hunk 2 above)
Comment (jhchee18) [reply src/main/java/seedu/application/ui/InterviewCard.java]: Ah thanks for pointing out XD they should be deleted

**Pull request merged**